### PR TITLE
support for record to table mapping config

### PR DESCRIPTION
**Pull request by patitapaban19**
Added support for the below configs:

1. "topic.my_topic.my_ks.my_table.mapping": "col1=key.f1, col2=value.f1, __ttl=value.f2, __timestamp=value.f3, col3=header.f1"
2. "topic.my_topic.my_ks.my_table.consistencyLevel": "LOCAL_ONE"
3. "topic.my_topic.my_ks.my_table.ttlSeconds": 1
4. "topic.my_topic.my_ks.my_table.deletesEnabled": "true"

Which will address the below:
- Display messages to determine the data structure of the topic messages.
- Create a topic-table map for Kafka messages that only contain a key and value in each record.
- For JSON fields, map individual fields in the structure to columns.
- Supports mapping individual fields from a Avro format field.
- Mapping a record with a key and Apache Kafka Struct value.
- Extract values from Kafka record header and write to the database table.
- Mapping messages to table that has a User Defined Type
- Ingest multiple topics and write to different tables.

Note: We are supporting only primitive data type to be mapped to table while extracting values from Kafka record header.

### CODE REVIEW

File: src/main/java/io/connect/scylladb/RecordConverter.java [hunk 1]
```diff
@@ -55,30 +58,56 @@
 
     protected abstract void setNullField(T result, String name);
 
-    public T convert(Object value) {
-        Preconditions.checkNotNull(value, "value cannot be null.");
+    public T convert(SinkRecord record, TopicConfigs topicConfigs, String operationType) {
+        Object recordObject = "delete".equals(operationType) ? record.key() : record.value();
```
Comment (tarzanek) [inline src/main/java/io/connect/scylladb/RecordConverter.java]: "delete" should be a static variable defined ideally in some common class
(and used below)

**Comment by tarzanek** [comment]
lgtm

**Comment by flucchese** [comment]
I'm missing a moredetailed description of what this PR is about. What problem/gap is it addressing? What is the strategy?

Notice also that the commited code uses a distinct tabbing standard, so many lines iwth no real modification are modified. Please stick to the original standard (4 spaces) and/or submit another PR with a global standard change.

**Comment by patitapaban19** [comment]
> I'm missing a more detailed description of what this PR is about. What problem/gap is it addressing? What is the strategy?
> 
> Notice also that the commited code uses a distinct tabbing standard, so many lines iwth no real modification are modified. Please stick to the original standard (4 spaces) and/or submit another PR with a global standard change.

@flucchese Have added the description for the pr. Please have a look.
Have used spacing as 2 for the connector but will raise a new pr with standard spacing after the above pr is merged.

**Comment by patitapaban19** [comment]
> My difficulty here is that there are several somehow unrelated changes squeezed into a single PR. Normally I would prefer incremental changes only, focused on one aspect of the code. Since I'm planning to conduct a major revision in the near future, i'm ok with these changes for now, making it clear that I haven't yet worked deeply into finding bugs or inconsistencies.

@flucchese This pr only contains the topic to mapping related configs support. We are raising pr with each related configs support is provided.

**Pull request merged**